### DISCUSSION TITLE
My work -> Repositories button does not work

### DISCUSSION THREAD

**Discussion by Hovhannisyan-K** [community/community#40914] [Mobile]
<details>
<summary>As soon as you click to open repos its showing something went wrong and a buttun"try again" most likely to update the page, but again it does the same</summary>
GitHub for iOS, Version 1.91.0 (85250348)
Device: iPhone 12 (iOS 16.1.2)
</details>

**Comment by mohamedzhioua** [comment]
maybe it's your connection

**Comment by ossanna16** [comment]
Hi there @Hovhannisyan-K and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by Hovhannisyan-K** [comment]


**Reply by mohamedzhioua** [reply]
@Hovhannisyan-K  
The only required fields are:

1 Owner
2 Repository Name
Once those 2 fields have been filled out the button will be enabled.

**Comment by jfullstackdev** [comment]
can we see a screenshot of these issues ?

**Reply by Hovhannisyan-K** [reply]
![IMG_1256](https://user-images.githubusercontent.com/44810857/208405932-a1092d86-ee29-4c0d-8fa2-47d1af173a50.PNG)
![IMG_1257](https://user-images.githubusercontent.com/44810857/208405941-bab55b9b-0190-4c18-8338-3af6a9b240ca.PNG)

**Reply by Hovhannisyan-K** [reply]
every time I get this as soon as I push the Repo button from main screen

**Reply by Hovhannisyan-K** [reply]
@jdevstatic

**Reply by jfullstackdev** [reply]
how about logging out then logging in ?

**Reply by Hovhannisyan-K** [reply]
Tried also

**Reply by Hovhannisyan-K** [reply]
Same issue remains
